core/build.gradle

```diff
@@ -47,6 +48,17 @@ dependencies {
 task installScanner(type: Exec) {
 
     executable "sh"
-    args "-c", "./../gradlew :scanner:publishToMavenLocal"
+    args "-c", "./../gradlew :scanner:publishToMavenLocal --rerun-tasks"
 }
 tasks.test.dependsOn("installScanner")
```

Comment:
Why do you need the `installScanner` task?  Can't you just do `tasks.test.dependsOn(':scanner:publishToMavenLocal')`?

Reply:
21af752c1c68d405fc599d54d1b6c572ce5ac2fd

Sure that works, thank you for the suggestion.